src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
While this might be OK I'd like to better understand why we need to move libraries to .NET 6.0? We generally try to target the lowest common framework for shared libraries to allow them to be used in the widest possible set of applications. It is mostly executables and test projects that need to target the latest .NET 6.0 runtime.

Reply:
My mistake, sorry! PR to revert:

- #5048

Reply:
It looks like this change may have caused a build issue for client libraries:

We're now seeing the following error building client libraries from within the VS2022 client:
Severity    Code    Description    Project    File    Line    Suppression State
Error    CS8032    An instance of analyzer Azure.ClientSdk.Analyzers.AsyncAnalyzer cannot be created from C:\Users\annelo\.nuget\packages\azure.clientsdk.analyzers\0.1.1-dev.20221229.1\analyzers\dotnet\cs\net6.0\Azure.ClientSdk.Analyzers.dll : Exception has been thrown by the target of an invocation..    Azure.Containers.ContainerRegistry (netstandard2.0)    C:\src\azure-sdk-for-net\sdk\containerregistry\Azure.Containers.ContainerRegistry\src\CSC    1    Active

What is the need for the analyzers package to target a higher framework?

Reply:
PR to revert:
- #5048